omas/omas_machine.py

```diff
@@ -16,6 +16,12 @@
 except:
     pass
 
+try:
+    from omas.machine_mappings import mast
+except ImportError:
+    pass
```

Comment:
Do you think it would be adding a warning message at least when MAST machine mappings are unavailable? This seems like it would be easy to miss